packages/updatePrescriptionStatus/src/updatePrescriptionStatus.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda"
+import {Logger} from "@aws-lambda-powertools/logger"
+import {injectLambdaContext} from "@aws-lambda-powertools/logger/middleware"
+import {DynamoDBClient, PutItemCommand} from "@aws-sdk/client-dynamodb"
+import {marshall} from "@aws-sdk/util-dynamodb"
+import middy from "@middy/core"
+import inputOutputLogger from "@middy/input-output-logger"
+import errorHandler from "@nhs/fhir-middy-error-handler"
+import {v4 as uuidv4} from "uuid"
+
+const logger = new Logger({serviceName: "updatePrescriptionStatus"})
+const client = new DynamoDBClient({region: "eu-west-2"})
+const tableName = process.env.TABLE_NAME
+
+interface DynamoDBItem {
+  RequestID: string;
+  PrescriptionID: string;
+  PatientNHSNumber: string;
+  PharmacyODSCode: string;
+  TaskID: string;
+  LineItemID: string;
+  TerminalStatusIndicator: string;
+  LastModified: string;
+  Timestamp: string;
+  RequestMessage: any;
+}
+
+const lambdaHandler = async (
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {
+  try {
+    console.log("Received event:", event)
+    const requestBody = JSON.parse(event.body || "")
+    console.log("Parsed request body:", requestBody)
```

Comment:
Extra log lines to denote progress are good but im not sure we should be logging the actual event/body as it contains NHS numbers

Reply:
Unnecessary logs have been removed